docs/tutorials/studio-tutorial.md

```diff
@@ -572,6 +572,15 @@ This panel lets you define tests for the services you write by specifying test d
 
 ## Create a Service Test 
 
+### Create connections test data
+Click the `generate` button to automatically generate test data if possible. Otherwise, please manually create test data.
+#### Data Type -- Relational
+Add relational data tables based on databases to which the service's mapping connects. It's unnecessary to provide all columns with test data. 
+The columns used in the service, however, require test data. Besides, the primary keys have to be distinct.
+![Create connections test data - Relational](../assets/create-a-relational-service-connection-test-data.gif)
+#### Data Type -- Data element
+One `Data element` could be used among different tests.
+![Create connections test data - Data Element](../assets/create-a-service-connection-test-with-data-element.gif)
 ### Relational
 
 ![Create a service test - Relational](../assets/create-a-service-test-relational.gif)
```

Comment:
Ca we add entires for External Format (M2M) and service store test data?

Reply:
+1